src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx

```diff
@@ -23,83 +23,65 @@ After setting up our Postfix integration, we give you a dashboard for your Postf
+    </Step>
+    <Step>
 ## Export Postfix data [#export]
 
-Use `pflogsumm` to export your Postfix metrics. `Pflogsumm` is a log analyzer for Postfix agent and designed to provide an overview of Postfix activity, with just enough detail to give the administrator a "heads up" for potential trouble spots.
+Use `pflogsumm` to export your Postfix metrics. `Pflogsumm` is a log analyzer for Postfix agent and designed to provide an overview of Postfix activity, with enough detail to give the administrator a "heads up" for potential trouble spots.
 
 Follow these steps to use `pflogsumm`.
 
-<CollapserGroup>
-  <Collapser
-    className="freq-link"
-    id="export"
-    title={<>Download <InlineCode>pflogsumm</InlineCode></>}
-  >
-  1. Run the following command to use `pflogsumm`:
+  1. Run the following commands to download and extract `pflogsumm`:
 
     ```shell
     wget https://jimsun.linxnet.com/downloads/pflogsumm-1.1.3.tar.gz
     ```
 
-  2. Extract the downloaded file.
-
     ```shell
     tar xvf pflogsumm-1.1.3.tar.gz
     ```
 
-  3. Navigate to the `pflogsumm` folder. 
+  2. Navigate to the `pflogsumm` folder. 
 
     ```shell
     cd pflogsumm-*.*.*
     ```
-  4. Copy the `pflogsumm.pl` file to the path `/usr/local/bin/pflogsumm` :
+
+  3. Copy the `pflogsumm.pl` file to the path `/usr/local/bin/pflogsumm` :
 
     ```shell
     sudo cp pflogsumm.pl  /usr/local/bin/pflogsumm
     ```
 
-  </Collapser>
-
-  <Collapser
-    className="freq-link"
-    id="export"
-    title={<>Run <InlineCode>pflogsumm</InlineCode> commands to get the Postfix metrics</>}
-  > 
+  4. Run the following `pflogsumm` commands to export the Postfix metrics:
 
     ```shell
     cat /var/log/mail.log | pflogsumm
     cat /var/log/mail.log | pflogsumm -d today
-    ```
-
-  </Collapser>
-</CollapserGroup>
-
-
+    ```    
+    </Step>
+    <Step>
 ## Configure NRI-Flex for Postfix
 
-Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:
+Flex, our application-agnostic monitoring tool, comes bundled with the New Relic infrastructure agent. To create a flex configuration file to report Postfix data follow these steps:
 
-1. Create a file named `postfix-flex-config.yml` in this path:
+1. Create a file named `postfix-flex-config.yml`:
 
   ```shell
-  /etc/newrelic-infra/integrations.d
+  touch /etc/newrelic-infra/integrations.d/postfix-flex-config.yml
   ```
 
-2. Update the file `postfix-flex-config.yml` with these configurations:
-      * `EVENT_TYPE`: You can consider `EVENT_TYPE` as a New Relic database table that you can query using NRQL.
-      * `COMMAND`: This contains the command which is used to print metrics on the terminal.
-
-  Once your configuration file is updated, it will look like below:
+2. Update the file `postfix-flex-config.yml`. You can use the configuration sample below, or create your own using the following pattern:
```

Comment:
```suggestion
2. Update the file `postfix-flex-config.yml`. We recommend you work off our configuration sample below:

EXAMPLE HERE

If you'd prefer to make your own, then refer to the snippet below. Make sure that your yaml file follows this pattern:

```yml

- event_type: A New Relic database table that you can query using NRQL.
    commands: The command used to print metrics on the terminal.
```

Reply:
although frankly i think this is even too verbose. customers can just refer to our provided snippet but the "make your own instructions" should prob just be AFTER the example with the explanations for the value they need to put in